openproblems/tasks/_batch_integration/batch_integration_graph/datasets/immune.py

```diff
@@ -0,0 +1,39 @@
+from .....data.immune_cells import load_immune
```

Comment:
Might be better to import starting from the module name instead of relative redirections, to make it more readable and robust.
e.g. 
```
from openproblems.data.immune_cells import load_immune
```

Reply:
sounds like a good idea. Are you keen to adapt this?